recipes/imagededup/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "imagededup" %}
+{% set version = "0.2.1" %}
+{% set sha256sum = "5bd41845550cf4c4dd4431a3932dcb8beca138ffee895cd1183c9d517ec9fb52" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - matplotlib
+    - cython >=0.29
+    - numpy <1.17
```

Comment:
Can you build with the conda-forge pins for numpy here.  Just remove the version constraint

Reply:
Okay used the conda-forge pinning instead and removed the constraint for numpy.

Reply:
There seems to be an error now in the Linux build?